repository_service_tuf/cli/admin/ceremony.py

```diff
@@ -343,48 +343,100 @@ def _configure_keys(
 ) -> Generator[RSTUFKey, None, None]:
     key_count = 1
     while key_count <= number_of_keys:
+        console.print(
+            f"\n🔑 Key {key_count}/{number_of_keys} [cyan]{role}[/]\n"
+        )
+
+        if role == Roles.ROOT.value and key_count == 1:
+            signing_key = "private"
+
+        elif role == "ONLINE":
+            signing_key = "public"
+        else:
+            signing_key = prompt.Prompt.ask(
+                "[cyan]Private[/] or [cyan]Public[/] key"
+                "\n- [cyan]private key[/] requires the file path and password"
+                "\n- [cyan]public info[/] requires the a key id and key hash"
+                "\n  tip: `rstuf key info` retrieves the public information"
+                "\nSelect to use [cyan]private key[/] or [cyan]public[/]?",
+                choices=["private", "public"],
+                default="public",
+            )
+
         key_type = prompt.Prompt.ask(
-            f"\nChoose {key_count}/{number_of_keys} [cyan]{role}[/] key type",
+            f"Select the [cyan]{role}[/]`s key type",
             choices=KeyType.get_all_members(),
             default=KeyType.KEY_TYPE_ED25519.value,
         )
-        filepath = prompt.Prompt.ask(
-            f"Enter {key_count}/{number_of_keys} the "
-            f"[cyan]{role}[/]`s private key [green]path[/]"
-        )
+        if signing_key == "private":
+            filepath = prompt.Prompt.ask(
+                f"Enter the [cyan]{role}[/]`s [green]private key path[/]"
+            )
 
-        colored_role = click.style(role, fg="cyan")
-        colored_pass = click.style("password", fg="green")
-        password = click.prompt(
-            f"Enter {key_count}/{number_of_keys} the "
-            f"{colored_role}`s private key {colored_pass}",
-            hide_input=True,
-        )
-        name = prompt.Prompt.ask(
-            "[Optional] Give a [green]name/tag[/] to the key",
-            default="",
-            show_default=False,
-        )
-        role_key: RSTUFKey = load_key(filepath, key_type, password, name)
+            password = click.prompt(
+                f"Enter the {role}`s private key password",
```

Comment:
You can make `role` colored again as I did for the metadata update here:
https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/59a98b8e502b62efb47f07e2ff53d20edb9e2e38/repository_service_tuf/cli/admin/metadata.py#L179